### PR TITLE
Docs italian version

### PR DESCRIPTION
**Pull request by AngyDev**
Related issue: #24391

**Description**

Add Italian translation of Extras

### CODE REVIEW

File: docs/api/it/extras/core/ShapePath.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+		<h3>[method:Array toShapes]( [param:Boolean isCCW], [param:Boolean noHoles] )</h3>
+		<p>
+		isCCW -- Modifica la modalità di generazione degli hole e dei solidi<br/>
+		noHoles -- Se generare o meno gli hole
+		</p>
+		<p>
+      Converte l'array [page:ShapePath.subPaths subPaths] in un array di Shape. Per impostazione predefinita
+      le forme solide sono definite in senso orario (CW) e i fori sono definiti in senso antiorario (CCW).
+      Se isCCW è impostato su true, le forme sono capovolte. Se il parametro noHoles è impostato su true,
```
Comment (DavideBruner) [inline docs/api/it/extras/core/ShapePath.html:86]:
```suggestion
      Se isCCW è impostato su `true`, le forme sono capovolte. Se il parametro noHoles è impostato su `true`,
```

File: docs/api/it/extras/curves/CatmullRomCurve3.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+		<h2>Proprietà</h2>
+    <p>Vedi la classe [page:Curve] per le proprità comuni.</p>
+
+		<h3>[property:Array points]</h3>
+		<p>L'array di punti [page:Vector3] che definisce la curva. Ha bisogno di almeno due entries.</p>
+
+		<h3>[property:Boolean closed]</h3>
+		<p>Quando questa proprietà viene impostata su true la curva ritornerà su se stessa.</p>
```
Comment (DavideBruner) [inline docs/api/it/extras/curves/CatmullRomCurve3.html:62]:
```suggestion
		<p>Quando questa proprietà viene impostata su `true` la curva ritornerà su se stessa.</p>
```

**Review by DavideBruner — COMMENTED** [review]
Overall LGTM. I do have few questions and suggestions (non-blocking tho).

**Comment by mrdoob** [comment]
Thanks!

**Comment by DreamingLeeds** [comment]
> Related issue: #24391
> 
> **Description**
> 
> Add Italian translation of Extras

222222222

**Comment by DreamingLeeds** [comment]
> > Related issue: #24391
> > **Description**
> > Add Italian translation of Extras
> 
> 222222222

11111111

**Pull request merged**